api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -72,6 +72,27 @@ message HTTPFaultConfig {
   }
 }
 
+// The configuration of a Redis fault.
+message RedisFaultConfig {
```

Comment:
Creating a new FaultConfig here. I think redis faults are different enough that it should have a different abstraction from HTTP faults. Some more discussions in https://docs.google.com/document/d/1LWVQsjkkbZx-K5Rc1-aJK5W6xwio4xrzAk-mOJz6b1Y/edit#heading=h.smkarzr1nskl.